WordPress/Classes/Utility/WPAuthTokenIssueSolver.h

```diff
@@ -0,0 +1,24 @@
+#import <Foundation/Foundation.h>
+
+typedef void(^WPAuthTokenissueSolverCompletionBlock)();
```

Comment:
Little nitpick, probably want to capitalize the `i` in `issue` so it's `WPAuthTokenIssueSolverCompletionBlock`